pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+log_format = %(asctime)s %(levelname)s %(message)s cirrus_daac_version: %(cirrus_daac_version)s, cirrus_core_version: %(cirrus_core_version)s, cumulus_version: %(cumulus_version)s, granule_name: %(granule_name)s,  workflow_execution_name: %(workflow_execution_name)s
+log_date_format = %Y-%m-%d %H:%M:%S
```

Comment:
I feel like there must be a way to accomplish whatever this is without setting the log format globally for all tests.